dlf/plugins/search/class.tx_dlf_search.php

```diff
@@ -537,11 +562,6 @@ public function main($content, $conf) {
 
 				}
 
-				// Add highlighting for fulltext.
-				$params['hl'] = 'true';
-
-				$params['hl.fl'] = 'fulltext';
-
```

Comment:
Does highlighting still work without using the `fastVectorHighlighter` when removing those lines?

Reply:
Yes, because those lines have been moved to tx_dlf_list->getRecord(). Highlights are only fetched for the record and not for the whole search result.